ginga/rv/plugins/LineProfile.py

```diff
@@ -275,45 +303,55 @@ def _slice(self, naxes, mk):
 
         # For axis > 3
         for i in range(2, naxes):
-            slice_obj[i] = int(round(self.image.revnaxis[i-2] + 1))
+            if i != i_sel:
+                slice_obj[i] = 0  # int(round(self.image.revnaxis[i-2] + 1))
```

Comment:
For a normal cube this would be expected, I think, since you are sending a "straight, perpendicular" ray through the cube to make the plot and so just changing the slice isn't going to change the pixels that the ray passes through.

The graph should update however if you change axes.

I need to look at the code more closely to reacquaint myself with what is going on there.  I didn't author that one and only sort of understand what it is supposed to be doing.  I now understand its usefulness, but given the wide variety of what the higher dimensions in these multidimensional objects *represents*, I think the hardest thing will be figuring out how to label the axes.  I suppose there should be some sort of quasi FITS "standard" that covers that...